docs/user-guide/volumes.md

```diff
@@ -541,6 +542,42 @@ A `PortworxVolume` can be dynamically created through Kubernetes or it can also
 
 More details can be found [here](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/portworx/README.md)
 
+### ScaleIO
+ScaleIO is a software-based storage platform that can use existing hardware to create clusters of scalable 
+shared block networked storage.  The ScaleIO volume plugin allows deployed pods to access existing ScaleIO 
+volumes (or it can dynamically provision new volumes for persistent volume claims, see 
+[ScaleIO Persistent Volumes](./persistent-volumes/index.md#scaleio)).
```

Comment:
Since this document has moved, you'll probably want to use a less relative URL for the link:

/docs/user-guide/persistent-volumes/#scaleio